.travis.yml

```diff
@@ -26,6 +26,8 @@ before_script:
   - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
 script:
   - pytest --doctest-modules || true
+  # The files below are expected to pass their doctests on Python 2 & 3.
+  - pytest --doctest-modules infogami/infobase/account.py infogami/infobase/bootstrap.py infogami/infobase/cache.py infogami/infobase/client.py infogami/infobase/common.py infogami/infobase/core.py infogami/infobase/dbstore.py infogami/infobase/infobase.py infogami/infobase/logger.py infogami/infobase/logreader.py infogami/infobase/lru.py infogami/infobase/readquery.py infogami/infobase/tests/pytest_wildcard.py infogami/infobase/utils.py infogami/infobase/writequery.py infogami/core/code.py infogami/core/helpers.py infogami/utils/app.py infogami/utils/i18n.py infogami/utils/storage.py infogami/infobase/common.py infogami/infobase/client.py infogami/infobase/dbstore.py infogami/infobase/lru.py infogami/infobase/readquery.py infogami/infobase/utils.py infogami/infobase/writequery.py
```

Comment:
Let's pull this out into a separate file (similar to https://github.com/internetarchive/openlibrary/blob/master/scripts/test-py3.sh ), with 1 test per line. This line will be a source of a bunch of conflicts as people fix up more failing tests, and start adding to it!